common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -599,6 +599,7 @@ class @Problem
         $p = $(element).find('span.status')
         `// Translators: the word unanswered here is about answering a problem the student must solve.`
         $p.parent().removeClass().addClass "unsubmitted"
+        $(element).find('.status .status-icon').remove()
```

Comment:
This remove is on change.  We need this so that when a user makes a change to a field it will delete the status icon.